src/containers/EditDonor/index.js

```diff
@@ -0,0 +1,145 @@
+import React, {Component, useState} from "react";
+import Button from "react-bootstrap/Button";
+import FormGroup from "react-bootstrap/FormGroup";
+import Form from "react-bootstrap/Form";
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
+import Card from "react-bootstrap/Card";
+import * as actions from "./actions";
+import FormControl from "react-bootstrap/FormControl";
+import {bindActionCreators} from "redux";
+import {connect} from "react-redux";
+
+function mapDispatchToProps(dispatch) {
+  return bindActionCreators({
+    getDonor: actions.getDonor,
+  }, dispatch);
+}
+
+export class EditDonor extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      data: {},
+      id: "",
+      name: "",
+      phone: "",
+      email: "",
+      address: "",
+      city: "",
+      state: "",
+      zipCode: "",
+      setEmail: "",
```

Comment:
Do we need `setEmail`?

Reply:
We do not need setEmail or setPassword, they came from CreateDonor which I copied most of the code from